aruco_detect/src/aruco_detect.cpp

```diff
@@ -273,6 +278,12 @@ void FiducialsNode::configCallback(aruco_detect::DetectorParamsConfig & config,
     detectorParams->polygonalApproxAccuracyRate = config.polygonalApproxAccuracyRate;
 }
 
+void FiducialsNode::ignoreCallback(const std_msgs::String& msg)
```

Comment:
Should set the param here so that the currently ignored fiducial list is visible outside the node,